Changelog.md

```diff
@@ -17,13 +17,15 @@
 ### Misc Changes
 
 
-## 0.28.1 -- 2023-03-19
+## 0.28.1 -- 2023-03-31
 
 ### Misc Changes
 
 - [#579]: `ElementWriter.write_inner_content` now uses a `FnOnce` instead of a more restrictive `Fn` closure
+- [#584]: Export EscapeError from the crate
```

Comment:
You should put changelog under **Unreleased** section instead of modifying existing release. Also add a backticks around the type:
```suggestion
- [#584]: Export `EscapeError` from the crate
```